File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/UiActions.java
package com.github.kotvertolet.youtubejextractor.models.test;

import com.google.gson.annotations.SerializedName;

public class UiActions{

	@SerializedName("hideEnclosingContainer")
	private boolean hideEnclosingContainer;

	public void setHideEnclosingContainer(boolean hideEnclosingContainer){
		this.hideEnclosingContainer = hideEnclosingContainer;
	}

	public boolean isHideEnclosingContainer(){
		return hideEnclosingContainer;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/UiActions.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import com.google.gson.annotations.SerializedName

class UiActions {
    @SerializedName("hideEnclosingContainer")
    var isHideEnclosingContainer = false
    override fun toString(): String {
        return "UiActions{" +
                "hideEnclosingContainer = '" + isHideEnclosingContainer + '\'' +
                "}"
    }
}